manifests/i/Instatus/Out/1.0.6/Instatus.Out.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Instatus.Out
+PackageVersion: 1.0.6
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://github.com/instatushq/out/releases/download/v1.0.6/Instatus.Out.Setup.1.0.6.exe
+  InstallerSha256: 59a9ffbdeafe8e3f87a0a114a758648c3c87d38a4be73c9eba4feeb51ef16a0d

```

```suggestion
  InstallerSha256: 59a9ffbdeafe8e3f87a0a114a758648c3c87d38a4be73c9eba4feeb51ef16a0d
  Scope: user
```
`C:\Users\%USERNAME%\AppData\Local\Programs\Instatus Out`